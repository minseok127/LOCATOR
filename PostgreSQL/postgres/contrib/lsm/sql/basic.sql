--
-- Test basic select, insert, delete, update operations
--

\c lsmtest

CREATE FOREIGN TABLE test(key TEXT, value TEXT) SERVER lsm_server;  

INSERT INTO test VALUES('YC', 'VidarDB');  
SELECT * FROM test;  

INSERT INTO test VALUES('California', 'Waterloo');  
SELECT * FROM test;  

DELETE FROM test WHERE key='California';  
SELECT * FROM test;  

UPDATE test SET value='VidarSQL';  
SELECT * FROM test;  

DROP FOREIGN TABLE test;  
