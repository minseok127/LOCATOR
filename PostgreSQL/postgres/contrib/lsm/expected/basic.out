--
-- Test basic select, insert, delete, update operations
--
\c lsmtest
CREATE FOREIGN TABLE test(key TEXT, value TEXT) SERVER lsm_server;  
INSERT INTO test VALUES('YC', 'VidarDB');  
SELECT * FROM test;  
 key |  value  
-----+---------
 YC  | VidarDB
(1 row)

INSERT INTO test VALUES('California', 'Waterloo');  
SELECT * FROM test;  
    key     |  value   
------------+----------
 YC         | VidarDB
 California | Waterloo
(2 rows)

DELETE FROM test WHERE key='California';  
SELECT * FROM test;  
 key |  value  
-----+---------
 YC  | VidarDB
(1 row)

UPDATE test SET value='VidarSQL';  
SELECT * FROM test;  
 key |  value   
-----+----------
 YC  | VidarSQL
(1 row)

DROP FOREIGN TABLE test;  
